src/screens/GroupSetting/hooks/useGroupSetting.js

```diff
@@ -28,6 +29,13 @@ const useGroupSetting = ({navigation, route}) => {
     setChangeName(true);
   };
 
+  React.useEffect(() => {
+    if (route?.params) {
+      setParticipants(route?.params?.rawJson?.channel?.members);
+      setCountUser(route?.params?.rawJson?.channel?.members?.length || 0);
+    }
+  }, [route]);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

This `useEffect` hook sets the `participants` and `countUser` states based on the `route` prop. However, it's not clear what happens when `route` changes after the component has mounted. If the `route` prop can change and you want to update the states when it does, this implementation is correct. But if `route` should not change or if changing it should not update the states, you should remove the dependency array so that the effect only runs once on mount.
```diff
-   }, [route]);
+   }, []);
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->